proposals/evaluation/uc-3-inheritance.md

```diff
@@ -83,11 +83,11 @@ If that actor makes a `GET` request to `<report.acp>` it will receive a graph is
 # Resource: </weekly-status/2021-04-28/report.acp>
 <#ac1>
   a acp:AccessControl ;
-  acp:apply </acp/research#p1> ;
-  acp:applyMembers </acp/research#p1> .
+  acp:apply </acp/research#p1> .
 ```
 
-Note that `</weekly-status/2021-04-28/report.acp>` is basically a copy of `</weekly-status/.acp>`. In the current proposal, all resources in ACP have their own associated effective Access Control Resource.
+Note that the triples in `</weekly-status/2021-04-28/report.acp>` are generated from the statements in `</weekly-status/.acp>`. 
+All resources in ACP have their own associated effective Access Control Resource.
```

Comment:
Does ACP here mean Access Control Protection or Policy?  If the first, it isn't the expansion used elsewhere nearby, so I would expand it and not use the acronym.  If the second, the sentence will need some rephrasing.

Reply:
ACP means "Access Control Policy" refering to the new spec in the making.